tests/org.jboss.tools.openshift.ui.bot.test/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenShift tooling integrations tests
+This plugin contains OpenShift 2 and OpenShift 3 tooling integrations tests. For execution of a specific tests see guidelines below.
+
+## OpenShift 2 integration tests
+Running OpenShift 2 tests successfully requires OpenShift 2 instance with an account. The account should allow to create more domains and should have at least 4 gears. Basic account on OpenShift online will fail on some tests because of domain count restriction and low amount of gears.
+
+## OpenShift 3 integration tests
+Running OpenShift 3 integrations tests requires OpenShift 3 instance. There are no further requirements as for OpenShift 2 tests.
```

Comment:
Nope, it means there are no further restriction or requirements on OpenShift account. OS 2 tests requires "upgraded" account with more domains and gears. OpenShift 3 tests run with any OpenShift 3 account. Maybe bad wording.